cli/cmd/root.go

```diff
@@ -26,15 +29,15 @@ var rootCmd = &cobra.Command{
 
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
-func Execute(ctx context.Context) {
-	err := runCmd(ctx, Version)
+func Execute(ctx context.Context, ver string, docsUrl string) {
+	err := runCmd(ctx, ver, docsUrl)
 	if err != nil {
 		os.Exit(1)
 	}
 }
 
-func runCmd(ctx context.Context, ver string) error {
-	v := version.Format(ver)
+func runCmd(ctx context.Context, ver string, docsUrl string) error {
+	v := cmdutil.FormatVersion(ver)
 	rootCmd.SetVersionTemplate(v)
```

Comment:
It can, but this is for just when user wan't use version as flag for root command like --version, we have separate impl for version command

Reply:
What I meant is that setting `rootCmd.Version` should be sufficient. `SetVersionTemplate` is for overriding the template, so I think it might not  be needed

Reply:
Ohh true thats not required